bundles/org.openhab.voice.mimictts/src/main/java/org/openhab/voice/mimic/internal/MimicTTSService.java

```diff
@@ -269,7 +292,26 @@ public AudioStream synthesize(String text, Voice voice, AudioFormat requestedFor
                             "Cannot get Content-Length header from mimic response. Are you sure to query a mimic TTS server at "
                                     + urlTTS + " ?");
                 }
-                return new InputStreamAudioStream(inputStreamResponseListener.getInputStream(), AUDIO_FORMAT, length);
+
+                InputStream inputStreamFromMimic = inputStreamResponseListener.getInputStream();
+                try {
+                    if (!config.workaroundServletSink) {
+                        return new InputStreamAudioStream(inputStreamFromMimic, AUDIO_FORMAT, length);
+                    } else {
+                        // Some audio sinks use the openHAB servlet to get audio. This servlet require the
```

Comment:
Just for my information, which ones ?

Reply:
I only reproduced the issue and tested my fix against the chromecast binding, but a user also reports error with webaudio and yamaha musiccast.
Nobody complains, but as far as I remember, the squeezebox binding also use the audio servlet.